orangecontrib/spectroscopy/tests/test_owspectra.py

```diff
@@ -146,6 +146,18 @@ def test_select_line(self):
             sa = out.transform(Domain([out.domain[ANNOTATED_DATA_FEATURE_NAME]]))
             np.testing.assert_equal(sa.X, 1)
 
+    def test_Label_peaks(self):
```

Comment:
Add peak label tests that tests on ugly data. Whenever we write code we have to make as few assumptions about the data as possible. If a user merged two datasets, wavenumbers would not be continuous and there are going to be many unknown values. So, test also with data that:
1) Contains unknowns, lots of them, also some rows and columns that are completely unknown.
2) Has a mixed order of wavenumbers.

The code will surely have problems with such data, because I saw amax used which does not account for NaNs.